memory/memory.go

```diff
@@ -12,3 +18,24 @@ func (self *Memory) Collect() (result interface{}, err error) {
 	result, err = getMemoryInfo()
 	return
 }
+
+// Get returns a Memory struct  already initialized
+func Get() (*Memory, error) {
```

Comment:
`Get()` is pretty ambiguous even when considering that the parent object should be a descriptor. What are your thoughts on maybe using a function name with a bit more context (or even something like `CollectV2()`)?